replica/database.hh

```diff
@@ -1182,6 +1184,8 @@ private:
             const query::clustering_row_ranges& rows,
             db::timeout_clock::time_point timeout) const;
 
+    timer<> _flush_timer;
+
```

Comment:
probably timer should be `timer<lowres_clock>`. It's more efficient and it shouldn't be a problem that flush is delayed by a task quota (a few ms at most).

Reply:
Done. I read comments in `timer.hh` but it's not always clear about efficiency.
For instance, `table::_off_strategy_trigger` is steady_clock_type timer. Why not lowres?

Reply:
> Done. I read comments in `timer.hh` but it's not always clear about efficiency. For instance, `table::_off_strategy_trigger` is steady_clock_type timer. Why not lowres?

I think that timer would be fine with lowres, since it doesn't require high resolution. steady_clock_type is the default, so there will be many places that picked the default but could have picked lowres instead. High resolution is required for critical paths where a bit of delay (even task quota) can lead to imprecision.

Reply:
I've got it, thanks. Since there are no detailed explanation of it the only way to understand how it works is to analize scylla code considering it as perfect and copy constructions where it's possible...